src/search_service/config/plugins/categories/taxonomy.json

```diff
@@ -1,11 +1,11 @@
 {
     "category": "taxonomy",
     "solr": {
-        "core": "taxonomy_ci",
+        "core": "taxonomy",
         "visible_fields":["taxonomy_id","scientific_name","scientific_lineage","rank","kingdom","domain","aliases","parent_taxon_ref","genetic_code","division_id","ws_ref"], 
         "sort_fields": ["taxonomy_id", "scientific_name", "domain", "rank", "kingdom"],
         "mapped_sort_fields": {"scientific_name": "scientific_name_sort"},
-        "facet_fields": ["parent_taxon_ref","domain"],
-        "query_string": "&fq=facet.limit=10&facet.mincount=1"
+        "facet_fields": [],
```

Comment:
Yeah.  For taxonomy, the faceting was blocking any result to be returned.  The front end will show the visible fields of all indexed taxon documents.